### ISSUE TITLE
JAWS html+ issue

### ISSUE THREAD

**Issue by stuartlamour** [andrewnicols/moodle-atto_htmlplus#9]
Hi Andrew,

University of Montana has found an issue with html+ : 

When I attempt to edit the HTML of any course content JAWS announces Atto as being blank or only containing a "-". Have verified the issue on 2 different Windows 10 PCs, using both JAWS 17 and 18, and using Internet Explorer, Firefox, and Chrome.

The issue seems to affect JAWS, NVDA appears not to be affected.

I have attached a screen capture recording of this issue as well for reference.
Replication instructions:
1. Login to the MOOC
2. Turn editing on
3. Edit any content - you can edit the content of a topic or a book chapter, it seems to make no difference.
4. Turn editing on
5. Select the HTML+ button
6. Set JAWS focus within the editor
7. Attempt to read the editor contents with the arrow keys.

You should find that JAWS doesn't read any of the HTML content. If you try to select all (Ctrl+A) JAWS seems to read out a single dash "-".

The inability to access the HTML source essentially prevents me from working on any of the updates we have planned for the MOOC, so we would appreciate your help resolving this issue as soon as possible.

Aaron M. Page

[JAWS_Atto_Issue_-_20170713_145401_42.mp4.zip](https://github.com/andrewnicols/moodle-atto_htmlplus/files/1148087/JAWS_Atto_Issue_-_20170713_145401_42.mp4.zip)
